rcaracaus/desarol
# 136: Footer Maxwidth

+ Added very basic footer width max-width
+ Note that this must be merged after the header, as there were changes in there I needed to utilize in this one

Steps to test
1) go to any page
2) see footer

Probably going to want to figure out a more solid design for this one, I don't know what social media (or other services) we may want to link to in the footer.
